src/dash-table/syntax-tree/lexeme/relational.ts

```diff
@@ -57,7 +57,7 @@ export const contains: IUnboundedLexeme = R.merge({
         op.toString().indexOf(exp.toString()) !== -1
     ),
     subType: RelationalOperator.Contains,
-    regexp: /^(contains)/i
+    regexp: /^(contains\s)/i
```

Comment:
An incomplete query would end with `contains` without any additional spaces, we want to match that too for proper error handling / user feedback

```suggestion
    regexp: /^(contains(\s|$))/i
```